### PR TITLE
test: Add GitPod support. Part I. 

### PR DESCRIPTION
**Pull request by nalmabrcom**
Completely refactored `lsp.spec.js`. CI will fail because `che-che4z` part should be merged first. 

Signed-off-by: Maryna Nalbandian <maryna.nalbandian@broadcom.com>

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
📟

File: tests/cypress/integration/LSP/lsp.spec.js [hunk 1]
```diff
@@ -497,9 +513,12 @@ context('This is a LSP spec', () => {
         hoverOverVariable(18, 'TOP-OF-PAGE', 'C01 IS TOP-OF-PAGE.');
         hoverOverVariable(29, 'TERMS-RECORD.', '01 TERMS-RECORD. 05 TERMS-KEY PIC 9(3). 05 FILLER PIC X(68).');
         hoverOverVariable(30, 'TERMS-KEY', '01 TERMS-RECORD. 05 TERMS-KEY PIC 9(3).');
+        cy.log('Seems an error in VSCODE');
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/lsp.spec.js:516]: Looks like debug logging. Can you delete it?

File: tests/cypress/integration/LSP/lsp.spec.js [hunk 2]
```diff
@@ -508,8 +527,9 @@ context('This is a LSP spec', () => {
   describe('TC319969 Provide the variables definition as documentation', () => {
     it(['smoke'], 'Checks variable definition into VSC autocomplete documentation.', () => {
       cy.openFile('HOVER.CBL');
-      varDifinitionAutocomplete('ADSF', '01 SOMETHING. 03 ADFSF OCCURS 30 TIMES. INDEXED BY INDX 05 OL-NO PIC X(8).');
+      varDifinitionAutocomplete('ADFSF', '01 SOMETHING. 03 ADFSF OCCURS 30 TIMES. INDEXED BY INDX 05 OL-NO PIC X(8).');
       varDifinitionAutocomplete('FILE-RECORD', '01 FILE-RECORD PIC X(113).');
+      cy.log('Seems an error in VSCODE');
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/lsp.spec.js:532]: And here too.

File: tests/cypress/integration/LSP/lsp.spec.js [hunk 3]
```diff
@@ -625,12 +642,13 @@ context('This is a LSP spec', () => {
...
       cy.goToLine(9)
         .getCurrentLineErrors({ expectedLine: 9 })
-        .getHoverErrorMessage()
+        .getHoverErrorMessage('EIBCALEN')
         .contains('Variable EIBCALEN is not defined');
       cy.getLineByNumber(6).type('LINKAGE SECTION.');
       cy.getLineByNumber(10).should('not.have.class', '.squiggly-error');
     });
     it('Define the constants as variables - check the go to definition', () => {
+      cy.log('Seems an error');
```
Comment (grianbrcom) [inline tests/cypress/integration/LSP/lsp.spec.js:651]: Again debug logging.

**Review by grianbrcom — CHANGES_REQUESTED** [review]

**Review by grianbrcom — APPROVED** [review]
:otter:

**Pull request merged**